Sources/PrimerSDK/Classes/User Interface/Root/PrimerRootViewController.swift

```diff
@@ -533,23 +532,25 @@ extension PrimerRootViewController {
             Primer.shared.primerRootVC?.showLoadingScreenIfNeeded(imageView: imgView, message: nil)
         }
         
-        paymentMethodTokenizationViewModel.didPresentPaymentMethodUI = {
-            
-        }
+        paymentMethodTokenizationViewModel.didPresentPaymentMethodUI = {}
         
         paymentMethodTokenizationViewModel.willDismissPaymentMethodUI = {
             Primer.shared.primerRootVC?.showLoadingScreenIfNeeded(imageView: imgView, message: nil)
         }
         
-//        paymentMethodTokenizationViewModel.tokenizationCompletion = { (tok, err) in
-//            if let err = err {
-//                Primer.shared.primerRootVC?.handle(error: err)
-//            } else {
-//                Primer.shared.primerRootVC?.handleSuccess()
-//            }
-//        }
+        //        paymentMethodTokenizationViewModel.tokenizationCompletion = { (tok, err) in
+        //            if let err = err {
+        //                Primer.shared.primerRootVC?.handle(error: err)
+        //            } else {
+        //                Primer.shared.primerRootVC?.handleSuccess()
+        //            }
+        //        }
         
-        paymentMethodTokenizationViewModel.startTokenizationFlow()
+        firstly {
+            paymentMethodTokenizationViewModel.startTokenizationFlow()
+        }
+        .done { _ in }
```

Comment:
Why is this empty?

Reply:
As we've turned into a `Promise` , I've added it to silence the warning 👇 @EvansPie 
<img width="1390" alt="Screen Shot 2022-06-14 at 12 18 57" src="https://user-images.githubusercontent.com/3584008/173554792-ad5843c8-d656-443d-90ec-8a3274610ba5.png">